runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8406,7 +8406,16 @@ class INTERPRETER_CLASS
 
 		J9JNIMethodID *methodID = (J9JNIMethodID *)(UDATA)J9OBJECT_U64_LOAD(_currentThread, memberNameObject, _vm->vmindexOffset);
 		J9ROMMethod *romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(methodID->method);
-		UDATA methodArgCount = romMethod->argCount;
+		UDATA methodArgCount = 0;
+		bool isVararg = J9_ARE_ALL_BITS_SET(romMethod->modifiers, J9AccVarArgs);
+
+		if (isVararg) {
```

Comment:
So this shouldn't be `isVarags` but something like `isSignaturePolymorphic`?  And varargs is an inexpensive proxy?